tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -55,7 +55,7 @@
  * </pre>
  */
 public class VoteResolver<T> {
-    public static final boolean DROP_HARD_INHERITANCE = true;
+    public static final boolean DROP_HARD_INHERITANCE = false;
```

Comment:
Not sure whether this makes a difference for CLDRModify, but anyway we don't want DROP_HARD_INHERITANCE = true until we're done with the vxml post-processing

Reply:
And we don't want it as is in v44 anyway.